azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/ProxyUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.utils;
+
+import com.microsoft.azure.common.logging.Log;
+import com.microsoft.azure.common.utils.TextUtils;
+import com.microsoft.azure.toolkit.lib.common.proxy.ProxyManager;
+import com.microsoft.azure.tools.auth.exception.InvalidConfigurationException;
+import com.microsoft.azure.tools.auth.util.ValidationUtil;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.apache.maven.execution.MavenExecutionRequest;
+import org.apache.maven.settings.Proxy;
+
+import java.util.List;
+
+public class ProxyUtils {
+    public static void initProxyManager(String httpProxyHost, String httpProxyPort, MavenExecutionRequest request) throws InvalidConfigurationException {
```

Comment:
how about to add it in ProxyManager together ?

Reply:
this is only used for maven, while ProxyManager is shared for all toolkits.